scripts/sizeDiffTable.swift

```diff
@@ -0,0 +1,69 @@
+import Foundation
+
+struct SizeDiff {
+    var oldSize: Int = 0
+    var newSize: Int = 0
+    var delta: Int {
+        return newSize - oldSize
+    }
+}
+var sizeDict: [String: SizeDiff] = [:]
+
+if CommandLine.arguments.count != 3 {
+    print("usage: swift sizeDiffTable.swift <path to old libFluentUI.a> <path to new libFluentUI.a>")
+} else {
+    let oldPath: String = CommandLine.arguments[1]
+    let newPath: String = CommandLine.arguments[2]
+
+    do {
+        parseArFor(path: oldPath, isOld: true)
+        parseArFor(path: newPath, isOld: false)
+
+        print("| File | Before | After | Delta |")
+        print("|------|--------|-------|-------|")
+
+        let sortedDict = sizeDict.sorted { abs($0.value.delta) > abs($1.value.delta) }
```

Comment:
abs - is this sorting the table by the absolute value of the diff? I might prefer large negatives at the bottom, so if you care about the biggest changes you go to the top for biggest positive change and bottom for biggest negative, rather than mixing negatives and positives together.

But not a dealbreaker.

Reply:
actually, swift string handles emojis very well. should we put positive or negative in the delta to make it visually good?

Reply:
Added different emojis for >250k, >100k, >0, and <0. We can move those numbers around if we want, though.